api/src/repl.ts

```diff
@@ -0,0 +1,32 @@
+import "tsconfig-paths/register";
+
+import { MikroORM } from "@mikro-orm/core";
+import { NestFactory } from "@nestjs/core";
+import { repl } from "@nestjs/core/repl";
+import { NestExpressApplication } from "@nestjs/platform-express";
+
+import { AppModule } from "./app.module";
+import { createConfig } from "./config/config";
+
+async function bootstrap() {
+    const config = createConfig(process.env);
+    const appModule = AppModule.forRoot(config);
+    const app = await NestFactory.create<NestExpressApplication>(appModule);
+    const replServer = await repl(appModule);
+
+    // preserve history between refreshes of the repl
+    replServer.setupHistory(".nestjs_repl_history", (err) => {
+        if (err) {
+            console.error(err);
+        }
+    });
+
+    // MikroORM setup
+    const orm = app.get(MikroORM);
+
+    // Context shortcuts
+    const replContext = replServer.context;
+    replContext.orm = orm;
+    replContext.em = orm.em;
```

Comment:
strictly speaking you are not setting up MikroORM anymore

```suggestion
    // Context shortcuts
    const replContext = replServer.context;
    replContext.orm = app.get(MikroORM);
    replContext.em = app.get(MikroORM).em;
```

Reply:
I agree with you and corrected that in the commit https://github.com/vivid-planet/comet-starter/pull/408/commits/a11ea948a6c926dc44a61ae85e8b790d47cd8137